client/app/scripts/components/terminal.js

```diff
@@ -198,15 +207,17 @@ export default class Terminal extends React.Component {
     ev.preventDefault();
     const paramString = JSON.stringify(this.props);
     clickCloseTerminal(this.getPipeId());
-    openNewWindow(`terminal.html#!/state/${paramString}`);
+
+    const minWidth = this.state.pixelPerCol * 80 + (8 * 2);
```

Comment:
I have a little laptop screen, it gets very small w/ dev tools open. So when the embedded term appears it is very small. My first thought is: lets pop that out and make it bigger! But when it pops out it is very small so I have to immediately resize it.

Popping out windows is like freeing them! 80 chars is the ideal min size i think.